scopes/harmony/diagnostic/diagnostic.ts

```diff
@@ -0,0 +1,3 @@
+export interface Diagnostic {
+  diagnosticFn: () => Object;
```

Comment:
add a name and description to the type.
So later on you can easily know what value belongs to what diagnositic